css/style.css

```diff
@@ -1,32 +1,41 @@
-/**
- * Beware that Qt Assistant (QTextBrowser) supports only a limited set of 
- * HTML/CSS: http://qt-project.org/doc/qt-5/richtext-html-subset.html
- */
```

Comment:
This file will not be pushed back, but it wouldn't make sense to add and use anything which cannot work reliably in Qt Assistant.